google/cloud/bigquery/client.py

```diff
@@ -1937,7 +1936,6 @@ def create_job(
             )
         elif "query" in job_config:
             copy_config = copy.deepcopy(job_config)
-            _del_sub_prop(copy_config, ["query", "destinationTable"])
```

Comment:
Given that we are no longer removing the property, we could just collapse this whole clause into:

```python
        elif "query" in job_config:
            query_job_config = google.cloud.bigquery.job.QueryJobConfig.from_api_repr(
                job_config
            )
```

Reply:
Good idea. https://github.com/googleapis/python-bigquery/pull/893